ui/components/forms/flavors/select/index.scss

```diff
@@ -22,3 +22,35 @@
     }
   }
 }
+
+#{$css-prefix}form-element__select-container {
+    position: relative;
+    .#{$css-prefix}select {
+        -moz-appearance: none;
+        -webkit-appearance: none;
+        appearance: none;
```

Comment:
Which autoprefixer is your team using? It seems `appearance` isn't supported by most tools using this parser: https://github.com/postcss/autoprefixer/issues/43